docs/source/tools/watcher.rst

```diff
@@ -98,3 +98,33 @@ The log file will contain lines such as the following:
 - The slave message ``smsg`` sent from BRISC to the other RISC Vs are all Go ``G``; ``D`` would indicate Done
 - The kernel IDs ``k_ids`` running are ``4`` on BRISC, ``3`` on NCRISC and ``5`` on TRISC; look further down the log file
   to see the names and paths to those kernels
+
+Asserts
+-------
+
+With watcher enabled, asserts can be added to kernel code. Tripping an assert will cause the kernel
+to hang and report which assert was tripped in the watcher log. When the watcher is disabled,
```

Comment:
"Asserts are supported in kernel code.  When the watcher is disabled, asserts will be compiled out.  When the watcher is enabled tripping an assert will cause the kernel to hang and report which assert was tripped in the watcher log."

Reply:
Updated in latest commit